diff -urN syscallent1.h syscallent1.h
--- syscallent1.h	Fri Aug  3 14:27:54 2001
+++ syscallent1.h	Mon Sep  9 16:37:19 2002
@@ -50,7 +50,7 @@
 	{ 3,	TF,	sys_chown,		"lchown"	}, /* 16 */
 #endif
 	{ 0,	0,	sys_break,		"break"		}, /* 17 */
-	{ 2,	TF,	sys_oldstat,		"oldstat"	}, /* 18 */
+	{ 2,	TF,	printargs,		"oldstat"	}, /* 18 */
 	{ 3,	0,	sys_lseek,		"lseek"		}, /* 19 */
 	{ 0,	0,	sys_getpid,		"getpid"	}, /* 20 */
 	{ 5,	TF,	sys_mount,		"mount"		}, /* 21 */
@@ -60,7 +60,7 @@
 	{ 1,	0,	sys_stime,		"stime"		}, /* 25 */
 	{ 4,	0,	sys_ptrace,		"ptrace"	}, /* 26 */
 	{ 1,	0,	sys_alarm,		"alarm"		}, /* 27 */
-	{ 2,	0,	sys_oldfstat,		"oldfstat"	}, /* 28 */
+	{ 2,	0,	printargs,		"oldfstat"	}, /* 28 */
 	{ 0,	TS,	sys_pause,		"pause"		}, /* 29 */
 	{ 2,	TF,	sys_utime,		"utime"		}, /* 30 */
 	{ 2,	0,	sys_stty,		"stty"		}, /* 31 */
@@ -116,7 +116,7 @@
 	{ 2,	0,	sys_setgroups,		"setgroups"	}, /* 81 */
 	{ 1,	0,	sys_oldselect,		"oldselect"	}, /* 82 */
 	{ 2,	TF,	sys_symlink,		"symlink"	}, /* 83 */
-	{ 2,	TF,	sys_oldlstat,		"oldlstat"	}, /* 84 */
+	{ 2,	TF,	printargs,		"oldlstat"	}, /* 84 */
 	{ 3,	TF,	sys_readlink,		"readlink"	}, /* 85 */
 	{ 1,	TF,	sys_uselib,		"uselib"	}, /* 86 */
 	{ 1,	TF,	sys_swapon,		"swapon"	}, /* 87 */

